hamsm_restarting/pull_sample_data.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+DATA_URL="https://zenodo.org/record/6471213/files/sample_data.tar.gz"
+
+echo "Downloading sample data from Zenodo ... "
+wget $DATA_URL
+
+bash untar.sh sample_data.tar.gz
```

Comment:
Not sure what's happening, but it would always fail the untarring at this point, even if it's done manually (without the untar script). 
```
tar: Error exit delayed from previous errors.
restart_initialization.json: Skipping hardlink pointing to itself: restart_initialization.json```

Reply:
It's choking on this weird link (that shouldn't be there), but it'll still successfully extract